src/routes/absences/ActionButtons.svelte

```diff
@@ -0,0 +1,4 @@
+<div class="fixed bottom-0 from-base-500 w-full bg-gradient-to-t to-transparent">
+	<button class="h-12">Ahoj</button>
+	<button>Ahoj</button>
```

Comment:
K čemu tohle je?

Reply:
To potom budou ty buttony ale to nechám na separátní PR